docs/device-and-supply-chain-modelling.md

```diff
@@ -0,0 +1,78 @@
+# Device and Supply Chain modelling
+
+## Intro
+This document sets out the accumulated knowledge and / or assumptions about:
++ the evidence available within a device which can be used for attestation
++ models for firmware release lifecycles 
++ observations on how multiple products may be released from a single source and the implications for provisioning data
+
+Why are these significant? The models here have driven the design of the Veraison system, in particular the way supply chain data is provisioned into the system and the Endorsement store is constructed to allow verification queries to access reference values to be used for evidence appraisal. Those design conclusions can be found in other documents.
+
+The Veraison project welcomes feedback from real world examples as to the accuracy of the models and assumptions used here. The project will be seeking confirmation on these models from industry partners. If you have views on these topics, please provide relevant commentary via a github issue or start a conversation on the Zulip channel.
+
+## Device evidence classification
+The evidence within an attestation report can be categorised into the following classes. These classes are not an exhaustive set of all types of evidence that may be found within a report but are those necessary for correlating evidence with endorsement / reference value queries.
+
+### Trust Anchor
+All devices are ASSUMED to contain a Trust Anchor (TA) or a descendant from a Trust Anchor. This is normally provisioned into the device at manufacturing time but may also be provisioned to the device on enrolling into a management structure. The TA takes the form of a key or a seed from which an Attestation signing key can be derived. The attestation report will contain information, referred to here as a Trust Anchor ID (TA-ID), which a Verifier can use to locate an endorsement providing information on the public value of the relevant key to use to validate the signature on a report. The Trust Anchor may have unique-to-device or group scope (i.e. multiple devices contain the same trust anchor). The latter case has been used as one solution for privacy protection use cases. 
+
+### Hardware Group
+A device MAY contain a Hardware Group ID that is presented in attestation evidence. This is an identifier, provisioned during manufacturing, which can be used to classify the SoC used within the device and will be the same for any devices built using that SoC. Although termed as a hardware ID, in fact this identifier will classify the immutable Root of Trust, being the un-modifiable and un-measured first boot component as well as the specific SoC hardware within the SoC. The identifier is a public value and has no security value, but aids classification for a verifier. If no Hardware Group ID is included within the attestation evidence then an equivalent value can be provisioned into the verifier as a correlation to the Trust Anchor ID.
```

Comment:
The last line is particularly not clear to me?
If no Hardware Group ID is included within the attestation evidence then an equivalent value can be provisioned into the verifier as a correlation to the Trust Anchor ID.

May be please clarify the usage ? What added value does the "equivalent provides"?

Reply:
This refers to an identifier value to be used in place of a group claim in evidence.  Tweaked wording in latest push.